STANDARDS.md

```diff
@@ -1,16 +1,66 @@
-# Dataset Standards
 
-This file outlines the standards and requirements needed for submitting a dataset to QCArchive.
+This file outlines the standards and data requirements needed for submitting a dataset to QCArchive.
 This ensures that we have a consistent data model for downstream processes.
 
-## Required fields 
+STANDARDS version: 3-DRAFT (2020-11-13)
+
+We distinguish between standards for the datasets (i.e. the actual data), and the standards for training force fields.
 
-Current list:
-* Ensure all submissions have cmiles, most important are mapped hydrogen smiles
-* Ensure the WBO is requested for all submissions, this should be included in the scf properties list using the flag `wiberg_lowdin_indices`
+# Dataset Standards
 
-## Best practices
-* If any calculations are to be redone from another collection, re-use the old input (coordinates, atom ordering etc) as this will avoid running the calculation again and will just create new references in the database to the old results and should help keep the cost of the calculations down.  
+## Each molecule must have the following information:
+- Canonical isomeric explict hydrogen mapped SMILES
```

Comment:
I'm thinking about an eventual future where we store CMILES _and_ InChI. This framing made me realize that we're expecting the CMILES in the current plan to do two things:

* Contain the graph molecule information that needs to be added to a QC representation to get something parameterizable.
* Be a lookup key for FINDING this particular molecule again in the future

I think CMILES is a really good tool for the first need. But I think it's bad at the second case (since chemically identical molecules could have different CMILES, so we have to record things like software provenance to make the lookup work). So, knowing that we may incorporate InChI in the future, we may not need to make our first attempt do this perfectly.

Reply:
Does InChi "solve" this issue (the same InChi always produced for chemically-identical molecules), or is it just a different system from mapped CMILES with its own (but different) edge cases?

Reply:
Yes it should solve this as inchi is a separate standard that all toolkits agree on and I think it would be great to include in the molecule as well.